pipeline-maven/pom.xml

```diff
@@ -559,10 +551,18 @@
                   <repository>localhost/pipeline-maven/maven-home</repository>
                 </configuration>
               </execution>
+              <execution>
+                <id>build-sshd-image</id>
+                <goals>
+                  <goal>build</goal>
+                </goals>
+                <phase>process-test-resources</phase>
+                <configuration>
+                  <contextDirectory>src/test/resources/org/jenkinsci/plugins/pipeline/maven/docker/SshdContainer/</contextDirectory>
+                  <repository>localhost/pipeline-maven/sshd</repository>
+                </configuration>
+              </execution>
```

Comment:
arf, I feared this ... the sshd-image MUST be declared first, to be build first, as others are built on top of it